algo/src/main/java/com/hartwig/serve/sources/ckb/curation/CkbCurator.java

```diff
@@ -86,4 +102,85 @@ private Variant curate(@NotNull Variant variant) {
 
         return curatedVariant;
     }
+
+    @NotNull
+    public Location curateFacilityName(@NotNull Location location) {
+        for (CkbFacilityCurationZipEntry facilityCurationZipEntry : facilityCurationZipEntries) {
+            if (containsWord(facilityCurationZipEntry.city(), location.city().toLowerCase())) {
+                String zip = location.zip() != null ? location.zip().toLowerCase().replaceAll("\\s", "") : "";
+                if ((facilityCurationZipEntry.zip().equals("")) || (zip.contains(facilityCurationZipEntry.zip()))) {
+                    return ImmutableLocation.builder().from(location).facility(facilityCurationZipEntry.curatedFacilityName()).build();
+                }
+            }
+        }
+
+        if (location.facility() != null) {
+            for (CkbFacilityCurationNameEntry facilityCurationNameEntry : facilityCurationNameEntries) {
+                if (containsWord(facilityCurationNameEntry.facilityName(), location.facility().toLowerCase()) && containsWord(
+                        facilityCurationNameEntry.city(),
+                        location.city().toLowerCase())) {
+                    return ImmutableLocation.builder().from(location).facility(facilityCurationNameEntry.curatedFacilityName()).build();
+                }
+            }
+        }
+
+        for (CkbFacilityCurationManualEntry facilityCurationManualEntry : facilityCurationManualEntries) {
+            if (equalStringsOrNull(location.facility(), facilityCurationManualEntry.facilityName()) && location.city()
+                    .equals(facilityCurationManualEntry.city()) && equalStringsOrNull(location.zip(), facilityCurationManualEntry.zip())) {
```

Comment:
Another minor but the if statement her is fairly complex. I'd personally split this up in multiple lines, something like:
```
hasMatchingFacilityName = ...
hasMatchingCity = ...
hasMatchingZip = ...
(if hasMatchingFacilityName && hasMatchingCity && hasMatchingZip) {
```

But up to you!